app/services/theme.js

```diff
@@ -16,10 +16,15 @@ export default Ember.Service.extend({
     store: Ember.inject.service(),
     session: Ember.inject.service(),
 
+    // If we're using a provider domain
+    isDomain: false,
```

Comment:
Not the biggest fan of this, could we possibly change it to something like `hasDomain`, `isProvider`, 'onProvider`, `onProviderDomain`? Don't feel strongly about any of these, but was initially confused by isDomain (although the comment is super helpful)